### PR TITLE
Lakehead - CEM-2597 Button

### PR DESCRIPTION
**Pull request by percypie**
Setup the initial framework for my ClickableButton component. Still needs onclick and hover functionality. Visuals for the default state should be complete.
Clickablebuttonsnip

### CODE REVIEW

**Comment by percypie** [comment]
pictures of the hovered and clicked button
Clickablebuttonsniphover
Clickablebuttonsnippress

**Comment by percypie** [comment]
Im going to check with my groupmates on colours when they are available, as I am very colourblind, but hopefully the colours are okay.

File: src/Containers/ClickableButton/ClickableButton.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        }
+        p:active{
+            background-color: ${theme.colors.clickedBackground};
+            color: ${theme.colors.clickedText};
+        }
+    `}
+`
+export default ClickableButton;
+
+//todo change from css to prop state for hover
```
Comment (ralph-dev) [inline src/Containers/ClickableButton/ClickableButton.tsx:43]: Waiting for todo

**Review by ralph-dev — CHANGES_REQUESTED** [review]

File: src/Containers/ClickableButton/ClickableButton.tsx (hunk 1 above)
Comment (percypie) [reply src/Containers/ClickableButton/ClickableButton.tsx:43]: in the meeting you mentioned that I should use the button in the react ui library as a guideline, but after looking at it, it is using css to change the button color as well, so im assuming that is not what you want?

Comment (ralph-dev) [reply src/Containers/ClickableButton/ClickableButton.tsx:43]: You can use the &:hover CSS if the button in react-ui is using it. but don't use it for onclick

**Comment by percypie** [comment]
Hey Ralph, I really wanted to figure this out on my own and I dont want to waste your time, but I cant figure out how to edit props using onclick actions and neither can my groupmates, is it possible you could give me something to point me in the right direction, I've tried following the documentation but I couldnt get it to work and there isnt a lot of stuff online for this sort of thing as far as I can tell. Sorry in advance and I hope to have something for you soon.

**Pull request closed without merging**